log10/load.py

```diff
@@ -246,28 +246,51 @@ def wrapper(*args, **kwargs):
                 }
                 for frame in current_stack_frame
             ]
+            assert (
+                kwargs is not None
```

Comment:
Just curious, when would this happen?

Reply:
We are checking a case like this, where `openai.Completino.create()` has no kwargs passed. 
```
        >>> from log10.load import log10
        >>> import openai
        >>> log10(openai)
        >>> response = openai.Completion.create()
        >>> print(response)
```

I found that openai api will return an InvalidRequestError if kwargs is empty. So seems redundant to have an assertion in our code. Thinking to remove this assertion.